src/BuildScriptGenerator/FinalStretchVersions.cs

```diff
@@ -4,9 +4,25 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Common
 {
     public static class FinalStretchVersions
     {
+        public const string FinalStretchDotNetCore11SdkVersion = "1.1.14";
+        public const string FinalStretchDotNetCore21SdkVersion = "2.1.818";
+        public const string FinalStretchDotNetCore22SdkVersion = "2.2.207";
+        public const string FinalStretchNetCoreApp10 = "1.0.16";
+        public const string FinalStretchNetCoreApp11 = "1.1.13";
```

Comment:
please update all of these stretch versions to be the final one: https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs#L546-L580